Backpack.podspec

```diff
@@ -14,6 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+define_subspec = Proc.new do |name, root_spec|
```

Comment:
very nice refactor!

Reply:
Too bad it doesn't actualyl work, had to remove it :(